### PR TITLE
Hide redis url if url length more than 40 characters

### PR DESCRIPTION
**Pull request by davydovanton**
Refer to #2411 
I think that best way to display redis url is hide this url if it's length more than 40 (or may be more?) characters.
Therefore I created toggle button with redis url and `(show)` text (also may be is better to translate this text for each locale file?).

### gif:
hfmvk16xaq

### CODE REVIEW

**Comment by seuros** [comment]
:+1:

File: web/views/_footer.erb [hunk 1]
```diff
@@ -6,7 +6,11 @@
             <p class="navbar-text" style="color:white;"><%= product_version %></p>
           </li>
           <li>
-            <p class="navbar-text redis-url">Redis: <%= redis_connection %></p>
+            <p class="navbar-text redis-url">
+              Redis:
+              <button class="redis-url__text"><%= redis_connection %></button>
+              <button class="redis-url__show">(show)</button>
```
Comment (seuros) [inline web/views/_footer.erb]: what about adding a label class ?
http://getbootstrap.com/components/#labels

Comment (davydovanton) [inline web/views/_footer.erb]: I don't knowю I think that it's unnecessary because it's button and label needs only for text information (status, etc).

**Comment by mperham** [comment]
I would rather the hostname be hidden with CSS if it's too long. I don't want to solution that involves JavaScript.

**Comment by davydovanton** [comment]
@mperham if you want use pure css I can add `overflow` for redis url. But this include some problems. For example you can check this PR #2393 where you can see what will happen for window users.
But anyway I think that long redis url it's big problem for web UI.

**Comment by mperham** [comment]
I don't understand the problem of using `overflow: hidden`.  Seems like that's the best solution.

**Comment by mperham** [comment]
PR #2393 switches back to `overflow: hidden` since 3.4.0 shipped with `overflow: scroll`.  I'm wondering if master is now correct again and fixes this issue.

**Comment by davydovanton** [comment]
@mperham okay, I close it

**Pull request closed without merging**